TL improvement (#527)
07th-mod/higurashi-console-arcs: Update/omot_006.txt

//rもはや説明するのも面倒くさい。…お袋の顔を立てる手前、無下にも扱えないし。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "もはや説明するのも面倒くさい。…お袋の顔を立てる手前、無下にも扱えないし。",

> 			NULL, "It would be too much of a pain to explain. ...For the sake of Mom's face, I can't abuse it.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It would be too much of a pain to explain. ...So Mom can save face, I won't say it plainly.", Line_Normal);
```

			NULL, " Kei-chan, you'll do fine.", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300281", 256, TRUE);
	OutputLine(NULL, "今はともかく何年かすれば、きっと鶴の一声で人が集まるようになるって！」",

> 			NULL, " Aside from the present, in just a few years, I'm sure people will gather on your orders too!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Rather than the present, in just a few years, I'm sure people will gather on your orders too!\"", Line_Normal);
```

	ClearMessage();

//rそう、ふと考え込みそうになったところへばしばしと乱暴に背中を叩きながら、まるで俺の不安を読みすかしたように、魅音が快活に笑ってみせた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そう、ふと考え込みそうになったところへばしばしと乱暴に背中を叩きながら、まるで俺の不安を読みすかしたように、魅音が快活に笑ってみせた。",

> 			NULL, "Though I was lost in thought, I came back to my senses when Mion suddenly patted my back roughly and laughed at me like she could read my mind.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Though I was lost in thought, I came back to my senses when Mion suddenly gave me a rough pat on the back and laughed at me like she could read my mind.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fcdb77>沙都子</color>", NULL, "<color=#fcdb77>Satoko</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 4, "ps3/s14/04/210400196", 256, TRUE);
	OutputLine(NULL, "「そ、その台詞はリボンと熨斗をつけてそっくりお返しいたしますわっ！」",

> 			NULL, "\"E-explaining it would be like combining ribbon and noshi!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"E-explaining it would be like combining ribbon and seaweed wrap!\"", Line_Normal);
```

	ClearMessage();

//圭一rvS14/01/210100273.「沙都子…リボンと熨斗は和洋折衷すぎて合わないぞ？」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100273", 256, TRUE);
	OutputLine(NULL, "「沙都子…リボンと熨斗は和洋折衷すぎて合わないぞ？」",

> 			NULL, "\"Satoko... wouldn't ribbon and noshi be a suitable fusion of Japanese and Western gifting styles?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Satoko... wouldn't ribbon and seaweed wrap be a suitable fusion of Japanese and Western gifting styles?\"", Line_Normal);
```

			NULL, "\"That's right, Satoko-chan?", Line_WaitForInput);
	ModPlayVoiceLS(4, 2, "ps3/s14/02/210200124", 256, TRUE);
	OutputLine(NULL, "　熨斗をつけて縛るのは水切りって紐で——」",

> 			NULL, " You can attach the noshi, then tie the mizukiri ribbon on top to...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " You can wrap it in the seaweed, then tie the ribbon on top to...\"", Line_Normal);
```

//rいや、熨斗の話を始めたのは沙都子だろ…？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いや、熨斗の話を始めたのは沙都子だろ…？",

> 			NULL, "Wait, wasn't Satoko the first to mention noshi...?", Line_Normal);

Reviewer: ```suggestion
			NULL, "Wait, wasn't Satoko the first to mention seaweed...?", Line_Normal);
```

			NULL, "\"Nyaaah~!", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s14/00/otab10002", 256, TRUE);
	OutputLine(NULL, "　普段のカワイイ店内で、はぁはぁしながら見つめるのも背徳感があっていいにょりが、明るい屋外で見るのもまた格別の果実にょりよ！！」",

> 			NULL, " It's nice to stare at them in the normal cute shop, nyah, but witnessing them in the bright outdoors is also like the special, rare fruit, nyah!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " It's nice to stare at them in the normal cute shop, nyah, but witnessing them in the bright outdoors is like a special, rare fruit, nyah!!\"", Line_Normal);
```

//r村長さんはそう言うと、有無を言わさず俺の手を力いっぱい握り締める。あまりの握力に手がつぶれるかと思ったほどだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "村長さんはそう言うと、有無を言わさず俺の手を力いっぱい握り締める。あまりの握力に手がつぶれるかと思ったほどだ。",
			NULL, "The mayor squeezed my hand with all his might as he said that. It felt like my hand was being crushed under his grip.", Line_Normal);
	ClearMessage();

//rに、しても…意中？　なんか気になることを今、この人は言わなかったか？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "に、しても…意中？　なんか気になることを今、この人は言わなかったか？",

> 			NULL, "Anyway... heart set on? Didn't this man say something anxious just now?", Line_Normal);

Reviewer: ```suggestion
			NULL, "Anyway... heart set on? Didn't this man say something troubling just now?", Line_Normal);
```

			NULL, " No need to be shy, Maebara-kun!", Line_WaitForInput);
	ModPlayVoiceLS(4, 18, "ps3/s14/18/213300013", 256, TRUE);
	OutputLine(NULL, "　魅音ちゃんの想い人ってことは、もう村中の人間が認めていることなんだからさ？」",

> 			NULL, " The whole village already knows you're Mion-chan's crush, you know?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " The whole village already knows you're Mion-chan's sweetheart, you know?\"", Line_Normal);
```

//r……確かに、あまりにも紛らわしい不用意な言い方。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "……確かに、あまりにも紛らわしい不用意な言い方。",

> 			NULL, "...Indeed, her speaking was careless enough to make misunderstandings.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Indeed, her words were careless enough to cause misunderstandings.", Line_Normal);
```

//r俺に振り返りながら走り出した魅音は、目の前に立つ女性に気づかず、激突した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "俺に振り返りながら走り出した魅音は、目の前に立つ女性に気づかず、激突した。",

> 			NULL, "Mion started running with her face toward me, so she inadvertently ran into a woman standing in front of her.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mion started running with her face still turned toward me, so she inadvertently ran into a woman standing in front of her.", Line_Normal);
```
